lib/setup-mandatory-files.sh

```
@@ -30,31 +42,54 @@ created=()
 skipped=()
 errors=()
 
-# --- File 1: CLAUDE.md ---
-claude_md="$PROJECT_DIR/CLAUDE.md"
-append_template="$TEMPLATE_DIR/CLAUDE-append.md"
+# Helper: write instruction file for a given platform
+# Args: $1=platform, $2=filename, $3=append_template, $4=full_template
+write_instruction_file() {
+  local plat="$1" fname="$2" append_tmpl="$3" full_tmpl="$4"
```

Shellcheck (0.11.0)

[warning] plat appears unused. Verify use (or export if used externally).

(SC2034)